extensions/wikia/AssetsManager/config.php

```diff
@@ -184,6 +184,14 @@
 	),
 );
 
+$config['adengine2_lookup_factory_js'] = array(
+	'skin' => [ 'oasis' ],
+	'type' => AssetsManager::TYPE_JS,
+	'assets' => array(
+		'//extensions/wikia/AdEngine/js/lookup/lookupFactory.js',
```

Comment:
What do you think about adding this dependency to `adengine2_tracking_js` instead of creating a new group? We already have bidders` dependencies there